Added copy edits (#1)
angrycub/katacoda-scenarios: nomad-reverse-proxy/intro.md

  distribute application traffic over many instances of the application, thus
  increasing its availability.

- **TLS termination/uplift** - Since the proxy server creates a new
  connection on behalf of the user, it can use a different TLS configuration for
  the client-side connection and the server connection. This fact allows
  operators to use a reverse proxy to terminate SSL at the proxy so that

>   application traffic is [[REPLACE COLLOQUIALISM (in the clear)]]. It also allows a connection to be

Author: ```suggestion
  application traffic is received unencrypted. It also allows a connection to be
```
